content/concepts/consensus-network/federated-sidechains.md

```diff
@@ -0,0 +1,91 @@
+
+_Door account_: An account controlled by the federators. There are two door accounts: one on the main chain and one on the side chain. Cross chain transactions are started by users sending assets to a door account. Main chain to side chain transactions cause the balance to increase on the main chain door account and the balance to decrease on the side chain door account. It is called a "door" because it is the mechanism to move assets from one chain to another - much like going between rooms in a house requires stepping through a door.
+
+_Triggering transaction_: A transaction that causes the federators to start the process of signing and submitting a new response transaction. For example, sending XRP to the main chain's door account is a triggering transaction that will cause the federators to submit a new transaction on the side chain.
+
+_Response transaction_: A transaction submitted by the federators in reaction to a triggering transaction. Note that triggering transaction and response transaction depends on context. Sending XRP from a door account to a user account is a response transaction when thinking about cross chain transactions. It is a triggering transaction when thinking about how to handle failed transactions.
+
+
+## How to Set Up a Federated Sidechain
+
+Federated Sidechains are currently available as an Engineering Preview so you can experiment and explore the potential of sidechains. You can connect sidechains to the XRP Ledger Testnet, Devnet, or Mainnet as soon as network servers upgrade to XRPL 1.8.
+
+**Caution:** You can connect sidechains to the XRP Ledger Mainnet to develop and test with small amounts; it is not recommended for production use cases until federated sidechains are available in a release. 
+
+Setting up a sidechain involves the following high-level steps:
+
+1. Clone the rippled sidechain branch: https://github.com/ripple/rippled/tree/sidechain.
+2. Write custom transactors for your sidechain. Note that this is an important and non-trivial task. Refer to the XRP Ledger Protocol Reference for information about transaction types.
+For example, the XRPL Labs Hooks project uses custom transactors. 
+3. Each sidechain federator has its own configuration file that must be updated to include the following information:  
+    - [sidechain] stanza - add details such as signing key, main chain account, and the main chain address (IP and port) to listen to. 
+    - [sidechain_assets] stanza - define assets that can be used for cross-chain transactions (XRP or IOU), exchange rate for the assets, and optional refund penalty to discourage transactions that may default.
+    - [sidechain_federators] stanza - list of federators public keys to be used for signing. This list is common for all federators on the sidechain.
+4. Setup “Door” accounts to enable cross chain transactions which involves the following tasks:
+    - Create and fund the door accounts. 
+    - Set the signers list for the door accounts.
```

Comment:
- Should be "SignerList", not "signers list"
- Link to [Set up multi-signing](set-up-multi-signing.html)